Make.inc

```diff
@@ -686,7 +686,11 @@ override ARCH := $(shell $(CC) -dumpmachine | sed "s/\([^-]*\).*$$/\1/")
 else
 ifneq ($(XC_HOST),)
 XC_HOST := $(ARCH)$(shell echo $(XC_HOST) | sed "s/[^-]*\(.*\)$$/\1/")
+ifneq (,$(filter ,$(findstring arm, $(ARCH)) $(findstring aarch64, $(ARCH))))
```

Comment:
What does filter do without a filter list? Is this the same as strip?

Reply:
On review I think this wouldn't  have worked. I've updated to the OR format you suggested for the other case. Thanks